lib/themes/dosomething/paraneue_dosomething/includes/preprocess/preprocess_node.inc

```diff
@@ -281,15 +281,16 @@ function paraneue_dosomething_preprocess_reportback_gallery(&$vars, $count = 6)
   drupal_add_js(
     [
       'dsKudosReactions' => [
+        'approved' => Kudos::getApprovedReactions(),
         'enabled' => !$disable_reactions,
-        'user' => isset($user->uid) ? $user->uid : null,
         'terms' => [
           // @TODO: Ideally this would call a function that retrieves all
           // kudos reaction terms and sets up the associative array, but
           // for the time being we are only using a single term, "heart",
           // so let's keep it simple ;)
           'heart' => dosomething_kudos_get_term_id_by_name('heart'),
         ],
+        'user' => isset($user->uid) ? $user->uid : null,
```

Comment:
We probably don't need a `user` value specifically scoped under `dsKudosReactions`. Would it be possible to use `drupal-user-id` meta tag, or maybe set a root-level `uid` property that everything can share?